### ISSUE TITLE
Inconsistent file names when 'Previewing in audacity' vs 'Open in audacity'

### ISSUE THREAD

**Issue by DilsonsPickles** [audacity/audacity#6050]
### Bug description

Once a project has been renamed on audio.com: 
<img width="756" alt="image" src="https://github.com/audacity/audacity/assets/111350316/238fa9e3-8d91-4f5d-b46a-fa823669bb8c">
It introduces some unplanned behaviour whereby the 'preview in Audacity' action will give the project the updated audio.com name. If however, the User presses 'Open in Audacity' action, the file will open with the original name.

### Steps to reproduce

1. Create and upload a project with multiple versions to audio.com
2. Got to audio.com and rename the project
3. Close all instances of the project
4. Press 'Preview in Audacity' on an older version and notice that the project name matches the name given on audio.com
5. Close the project preview
6. Now press 'Open in Audacity' and observe the original project name that was entered on initial upload

### Expected behavior

If possible, when the User renames the project in audio.com that name should be used consistently from then on.

### Audacity Version

current master / alpha / nightly build

### Operating system

macOS

**Comment by dozzzzer** [comment]
Duplicate of https://github.com/audacity/audacity/issues/5992

**Comment by crsib** [comment]
I disagree that this is a duplicate. Last Wed, we agreed that the name should be different, and @LWinterberg planned to think about it.
